### ISSUE TITLE
Leaflet map on android 8.0.0 takes more than 10 seconds to load

### ISSUE THREAD

**Issue by lentyaishe** [Leaflet/Leaflet#6196]
Hello,

We're using leaflet map component for quite a while and this is a great peace of code. Thank you!

Currently we ran into a problem with the newest android version 8.0.0. Up until now our application was running on android versions 5 till 7 but our customers started to upgrade their devices to android v.8.0.0.

We use cordova environment.

The issue is that the `load` event of the `map` object occurs after 10.7 seconds on android 8.0.0 every time the screen with the map is opened. The same does not happen on older versions of android or iOS: the same event is fired within 1.5-2 seconds.

Leaflet version we're using is `1.0.3` but we tried it with the latest `1.3.1` and the behavior is the same.

To summarize:
- Leaflet version: 1.0.3 (or the latest)
- Browser: cordova (android v.6.0.0)
- platform: android v.8.0.0
- Steps to reproduce: basic map page, add event listener for `map.on("load", ...` and check how much time will pass till the map loads.

Thanks in advance

**Comment by IvanSanchez** [comment]
Hi, and thanks for taking the time to open a bug report in Leaflet.

However, in this repository we only handle bugs in "[vanilla](http://vanilla-js.com/)" Leaflet. This means that we do **not** handle bugs which are specific to frameworks such as:
* React
* Ractive
* AngularJS
* Bootstrap
* Leaflet for R
* Joomla
* Wordpress
* Ionic
* Cordova
* Vue
* Electron
* Polymer
* TypeScript
* ...etc

Please understand that we only have the time and energy to test Leaflet in plain web browsers.

Please try to either reproduce the bug without using any frameworks, or submit a bug to the appropiate repo.
